src/ec/packages/ec-component-site-header-standardised/site-header-standardised.html.twig

```diff
@@ -0,0 +1,316 @@
+              icon: {
+                path: _icon_file_path,
+                type: 'general',
+                name: 'log-in',
+                size: 's'
+              }
+            } only %}
+            {{- login_toggle.label_not_logged -}}
+          </a>
+        {% endif %}
+        </div>
+      {% endif %}
+      {% if _language_selector is not empty %}
+        <a
+          class="ecl-link ecl-link--standalone ecl-site-header-standardised__language-selector"
+          href="{{ _language_selector.href }}"
+          data-ecl-language-selector
+        >
+          <span class="ecl-site-header-standardised__language-icon">
+            {% include '@ecl-twig/ec-component-icon/icon.html.twig' with {
+              icon: {
+                path: _icon_file_path,
+                type: 'general',
+                name: 'language',
+                size: 's'
+              }
+            } only %}
+            <span class="ecl-site-header-standardised__language-code">{{ _language_selector.code }}</span>
+          </span>
+          {{- _language_selector.label -}}
+        </a>
+      {% endif %}
+        <div class="ecl-site-header-standardised__search-container">
+        {% if _search_toggle is not empty %}
+          <a
+            class="ecl-link ecl-link--standalone ecl-site-header-standardised__search-toggle"
+            href="{{ _search_toggle.href }}"
+            data-ecl-search-toggle="true"
+            aria-controls="search-form-id"
+            aria-expanded="false"
+          >
+          {%- include '@ecl-twig/ec-component-icon/icon.html.twig' with {
+            icon: {
+              name: 'search',
+              path: _icon_file_path,
+              type: 'general',
+              size: 's',
+              extra_attributes: [
+               { name: 'aria-hidden', value: true }
+              ]
+            }
+          } only -%}
+          {{- _search_toggle.label -}}
+          </a>
+        {% endif %}
+          {% include '@ecl-twig/ec-component-search-form/search-form.html.twig' with _search_form only %}
+        </div>
+      </div>
+    </div>
+  </div>
+{% if banner_top is defined and banner_top is not empty %}
+  <div class="ecl-site-header-standardised__banner-top">
+    <div class="ecl-container">{{ banner_top|raw }}</div>
+  </div>
+{% endif %}
+{% if banner is defined and banner is not empty %}
+  <div class="ecl-site-header-standardised__banner">
+    <div class="ecl-container">
+      <a
+      class="ecl-link ecl-link--standalone ecl-site-header-standardised__menu-toggle"
+      href="#"
+      >
+        {%- include '@ecl-twig/ec-component-icon/icon.html.twig' with {
+            icon: {
+              name: 'hamburger',
+              path: _icon_file_path,
+              type: 'general',
+              size: 's',
+              extra_attributes: [
+               { name: 'aria-hidden', value: true }
+              ]
+            }
+          } only -%}
+        Menu
```

Comment:
extra space after Menu label in the rendered markup

Reply:
Uff, this is hardcoded in ecl, but it's the label shown in mobile... I added an input variable for this, but just for the sake of this iteration, this will change most likely....

Reply:
ok